plans/podman.fmf

```diff
@@ -0,0 +1,32 @@
+discover:
+    how: fmf
+execute:
+    how: tmt
+environment:
+    PODMAN: /usr/bin/podman
+    PODMAN_REMOTE: /usr/bin/podman-remote
+    PODMAN_TESTING: /usr/bin/podman-testing
+    QUADLET: /usr/libexec/podman/quadlet
+prepare:
+    how: shell
+    script: |
+        rm -f /etc/yum.repos.d/tag-repository.repo
+
+/upstream:
+    summary: Run tests on upstream PRs
+    discover+:
+        filter: tag:upstream
+    adjust+:
+        enabled: false
+        when: revdeps == yes and initiator is not defined or initiator != packit
```

Comment:
shouldn't that be "no"? This could be why the [cockpit revdeps test](https://artifacts.dev.testing-farm.io/3c475b4f-feff-4e73-8ca8-2d45e93ccf5e/) runs this podman plan.

TBH there is a *lot* of tmt stuff going on here, and I'm not familiar enough with it to see where exactly things are going wrong :cry:

Reply:
should've been an `or` instead of `and` . Fixed in the latest run and no podman job in cockpit anymore. https://artifacts.dev.testing-farm.io/0cb7f3da-06fc-489d-b6a3-528c0abd78ff/ . Thanks for looking into this.

Reply:
oh, of course, this is "enabled: false". :see_no_evil: Thanks!